tv_apps/browser/js/sync/settings.js

```diff
@@ -165,7 +165,7 @@
           // but we only want to do that if its enabled via user action
           // (and not because it is already enabled from a previous run).
           if (this.state === 'enabling') {
-            Settings.show();
+            Settings.show.bind(Settings)();
```

Comment:
It does :) Not binding Settings here will make `this` at [1] not be Settings and so it will throw.

You have to login into Sync from the toolbar to see this error in the console.

[1] https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/browser/js/settings.js#L171